.github/workflows/release_windows_pytorch_wheels.yml

```diff
@@ -78,11 +78,23 @@ permissions:
 
 jobs:
   release:
-    name: Release | ${{ inputs.amdgpu_family }} | py ${{ matrix.python_version }}
+    name: Release | ${{ inputs.amdgpu_family }} | py ${{ matrix.python_version }} | torch ${{ matrix.pytorch_version }}
     strategy:
       fail-fast: false
       matrix:
         python_version: ["3.11", "3.12", "3.13"]
+        # TODO: switch to 'nightly' to match our Linux workflows?
+        # TODO(#1677): Get 2.8 and 2.9 builds working and enable them
+        # pytorch_version: ["release/2.8", "release/2.9", "main"]
+        pytorch_version: ["main"]
```

Comment:
```suggestion
        pytorch_version: ["nightly"]
```

Reply:
I think the switch from main to nightly switch should be separate to this PR, so it is fine to leave as a TODO.